src/utils/assets.js

```diff
@@ -11,7 +11,7 @@
 /**
  * Utility functions for working with Assets
  */
-import { baseUrl } from 'docc-render/utils/theme-settings';
+const { baseUrl } = window;
```

Comment:
Why the change?

Reply:
Because it gives a "Dependency cycle via docc-render/utils/url-helper" ESLint error.

The reason is because `assets` imports `theme-settings`, `theme-settings` imports `url-helper` and `url-helper` imports `assets` ♻️
I stopped the cycle by not importing `theme-settings` into `assets`